doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,840 @@
+    This ensures that API user is aware of the error.
+
+Common Application Framework
+----------------------------
+
+|no_changes_yet_note|
+
+Debug libraries
+---------------
+
+|no_changes_yet_note|
+
+DFU libraries
+-------------
+
+|no_changes_yet_note|
+
+Gazell libraries
+----------------
+
+|no_changes_yet_note|
+
+Security libraries
+------------------
+
+|no_changes_yet_note|
+
+Modem libraries
+---------------
+
+* :ref:`modem_key_mgmt` library:
+
+  * Updated to handle generic CME errors from all ``AT%CMNG`` commands.
+
+Multiprotocol Service Layer libraries
+-------------------------------------
+
+|no_changes_yet_note|
+
+Libraries for networking
+------------------------
+
+|no_changes_yet_note|
+
+Libraries for NFC
+-----------------
+
+|no_changes_yet_note|
+
+nRF RPC libraries
+-----------------
+
+* Added the :ref:`nrf_rpc_dev_info` library for obtaining information about a device connected through the :ref:`nrfxlib:nrf_rpc`.
+
+Other libraries
+---------------
+
+|no_changes_yet_note|
+
+Security libraries
+------------------
+
+|no_changes_yet_note|
+
+Shell libraries
+---------------
+
+|no_changes_yet_note|
+
+Libraries for Zigbee
+--------------------
+
+|no_changes_yet_note|
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* :ref:`nrf_desktop_config_channel_script` Python script:
+
+  * Added support for pure ED25519 signature (used by nRF54L based devices that enable MCUboot hardware cryptography).
+    This requires using ``imgtool`` supporting pure ED25519 signature that can be installed from ``sdk-mcuboot`` repository.
+    See the script documentation page for details.
+
+Integrations
+============
+
+This section provides detailed lists of changes by :ref:`integration <integrations>`.
+
+Google Fast Pair integration
+----------------------------
+
+|no_changes_yet_note|
+
+Edge Impulse integration
+------------------------
+
+|no_changes_yet_note|
+
+Memfault integration
+--------------------
+
+|no_changes_yet_note|
+
+AVSystem integration
+--------------------
+
+|no_changes_yet_note|
+
+nRF Cloud integration
+---------------------
+
+|no_changes_yet_note|
+
+CoreMark integration
+--------------------
+
+|no_changes_yet_note|
+
+DULT integration
+----------------
+
+|no_changes_yet_note|
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``a4eda30f5b0cfd0cf15512be9dcd559239dbfc91``, with some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+|no_changes_yet_note|
```

Comment:
Anything to add here?

Reply:
Added option which allows to selects the number of KMU key slots (also known as generations) to use when verifying an image. see ``CONFIG_BOOT_SIGNATURE_KMU_SLOTS MCUboot's Kconfig option``.